cmd/kogito/command/install/trusty.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
```

Comment:
Don't worry copying and pasting the headers, `make test` or `make vet` would add automatically for you with the correct year. ;)